examples/models/voxtral/multimodal.cpp

```diff
@@ -319,7 +319,7 @@ int32_t main(int32_t argc, char** argv) {
   // Create multimodal runner
   std::unique_ptr<::executorch::extension::llm::MultimodalRunner> runner =
       ::executorch::extension::llm::create_multimodal_runner(
-          model_path, std::move(tokenizer), data_path);
+          model_path, std::move(tokenizer), data_path, Module::LoadMode::Mmap);
```

Comment:
lol so we were loading from disk -> writing to heap -> writing to disk -> loading temp from disk -> writing to cuda.